dns/zone-configs/kubernetes.io._0_base.yaml

```diff
@@ -18,6 +18,10 @@
       preference: 10
     - exchange: alt4.aspmx.l.google.com.
       preference: 10
+  - type: CAA # Which CA's can issue certs for the domain
```

Comment:
https://sslmate.com/caa/about describes how CAA resolution is done, so we only need CAA records for the root of each domain.  We should comment that, probably.